library/notary

```diff
@@ -1,8 +1,12 @@
 # maintainer: David Lawrence <david.lawrence@docker.com> (@endophage)
 
-server: git://github.com/docker/notary-official-images@5fbf22ada9e5167fe4e9340ac947e8b3110aceba notary-server
+server: git://github.com/docker/notary-official-images@v0.5.0 notary-server
+server-0.5.0: git://github.com/docker/notary-official-images@v0.5.0 notary-server
+signer: git://github.com/docker/notary-official-images@v0.5.0 notary-signer
+signer-0.5.0: git://github.com/docker/notary-official-images@v0.5.0 notary-signer
+server-0.4.2: git://github.com/docker/notary-official-images@v0.4.2 notary-server
+signer-0.4.2: git://github.com/docker/notary-official-images@v0.4.2 notary-signer
```

Comment:
Ouch, yeah, I can understand for docker/docker -- there it makes sense to declare a relationship between Docker and Notary based on version numbers, but here we prefer explicit commits to ensure that we're building exactly what was intended and that it hasn't changed in between unintentionally or maliciously (since this is really a "deployment" manifest, not a dependency relation). :disappointed:

Does that make more sense?